dataworks-aws-ucfs-claimant-interface.tf

```diff
@@ -0,0 +1,56 @@
+resource "github_repository" "dataworks_aws_ucfs_claimant_interface" {
+  name             = "dataworks-aws-ucfs-claimant-interface"
```

Comment:
Not sure about this name

Reply:
What name would you suggest @dan-hill2802? Seems to make sense to me and follow the other lambda -> RDS pattern we have?